assets/training/aoai/proxy_components/pipeline_components/aoai_finetune/spec.yml

```diff
@@ -0,0 +1,112 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: aoai_finetune
```

Comment:
Let us add pipeline suffix to name for easy identification

Reply:
fixed